src/utils.jl

```diff
@@ -248,12 +248,16 @@ end
 is_leaf(frame::Frame) = frame.callee === nothing
 
 function is_vararg_type(x)
-    if isa(x, Type)
-        (x <: Vararg && !(x <: Union{})) && return true
-        if isa(x, UnionAll)
-            x = Base.unwrap_unionall(x)
+    if isa(Vararg, Type)
```

Comment:
```suggestion
    @static if isa(Vararg, Type)
```

Reply:
Any functional reason to prefer `@static` here, or is it mostly stylistic?

Reply:
yeah, it also may make it clearer that this branching because of version compatibility when we read this code later.